enderio-machines/enderio-machines.gradle

```diff
@@ -1,4 +1,31 @@
+plugins {
+    id "io.freefair.lombok" version "6.1.0-m3"
```

Comment:
Why has lombok showed its face again lmao

Reply:
Some git stuff, seemed to have not merged properly